internal/app/machined/pkg/runtime/v1alpha1/v1alpha1_sequencer_tasks.go

```diff
@@ -81,41 +79,45 @@ import (
 	"github.com/siderolabs/talos/pkg/version"
 )
 
-// SetupLogger represents the SetupLogger task.
-func SetupLogger(runtime.Sequence, any) (runtime.TaskExecutionFunc, string) {
+// WaitForUSB represents the WaitForUSB task.
+func WaitForUSB(runtime.Sequence, any) (runtime.TaskExecutionFunc, string) {
 	return func(ctx context.Context, logger *log.Logger, r runtime.Runtime) (err error) {
-		machinedLog, err := r.Logging().ServiceLog("machined").Writer()
+		// Wait for USB storage in the case that the install disk is supplied over
+		// USB. If we don't wait, there is the chance that we will fail to detect the
+		// install disk.
+		file := "/sys/module/usb_storage/parameters/delay_use"
+
+		_, err = os.Stat(file)
```

Comment:
Minor nit: if you do `:=` here and below linter probably will not complain since you do not have any inner `err`s.

Reply:
yep, this is copy-pasted... I will drop named return args